app/models/manageiq/providers/azure/cloud_manager/refresh_parser.rb

```diff
@@ -125,7 +129,9 @@ def get_resource_status_message(resource)
 
       def get_stack_templates
         # download all template uris
+        _log.info("Retrieving templates...")
         @template_uris.each { |uri, template| template[:content] = download_template(uri) }
+        _log.info("Retrieving templates...Complete - Count [#{@template_uris.count}]")
 
```

Comment:
@agrare - The downloading of orchestration templates is expensive, can we monitor the memory usage there too?

Reply:
@bronaghs oh yeah definitely, I missed this one because it didn't use the usual inv collect method.